debian/eos-updater.install

```diff
@@ -17,8 +19,10 @@ usr/share/eos-updater/eos-updater.conf
 usr/share/eos-updater/eos-update-server.conf
 usr/share/man/man5/eos-autoupdater.conf.5
 usr/share/man/man5/eos-updater.conf.5
+usr/share/man/man5/eos-updater-flatpak-autoinstall.d.5
 usr/share/man/man5/eos-update-server.conf.5
 usr/share/man/man8/eos-autoupdater.8
-usr/share/man/man8/eos-updater-avahi.8
-usr/share/man/man8/eos-update-server.8
 usr/share/man/man8/eos-updater.8
+usr/share/man/man8/eos-update-server.8
+usr/share/man/man8/eos-updater-avahi.8
```

Comment:
Why reorder the files in the list here? Doesn’t seem like it’s moving to be more alphabetically ordered.